kbncontent.go

```diff
@@ -191,9 +189,14 @@ func (v VisualizationDescriptor) Title() string {
 
 func deserializeSubPaths(doc objx.Map) error {
 	jsonPaths := []string{
+		"attributes.kibanaSavedObjectMeta.searchSourceJSON",
+		"attributes.mapStateJSON",
 		"attributes.uiStateJSON",
 		"attributes.visState",
-		"attributes.kibanaSavedObjectMeta.searchSourceJSON",
+		"embeddableConfig.attributes.kibanaSavedObjectMeta.searchSourceJSON",
+		"embeddableConfig.attributes.mapStateJSON",
+		"embeddableConfig.attributes.uiStateJSON",
+		"embeddableConfig.attributes.visState",
```

Comment:
Adding the paths with the `embeddableConfig` prefix. Actually in all the test cases these paths are prefixed with `embeddableConfig`, so not sure if we need the others :thinking: